src/VisualStudio/Core/Test.Next/Remote/SerializationValidator.cs

```diff
@@ -32,7 +32,7 @@ public override async ValueTask<T> GetAssetAsync<T>(Checksum checksum, Cancellat
                 => await _validator.GetValueAsync<T>(checksum).ConfigureAwait(false);
         }
 
-        internal sealed class ChecksumObjectCollection<T> : IEnumerable<T> where T : ChecksumWithChildren
+        internal sealed class ChecksumObjectCollection<T> : IEnumerable<T>
```

Comment:
test helper.  this was benefitting from a generic way to process a few node types.  However, i preferred just making the typing explicit.  handwriting it was not a problem.